lib/matplotlib/font_manager.py

```diff
@@ -1304,16 +1323,51 @@ def findfont(self, prop, fontext='ttf', directory=None,
         rc_params = tuple(tuple(rcParams[key]) for key in [
             "font.serif", "font.sans-serif", "font.cursive", "font.fantasy",
             "font.monospace"])
+
+        if not isinstance(prop, FontProperties):
+            prop = FontProperties._from_any(prop)
+
         return self._findfont_cached(
             prop, fontext, directory, fallback_to_default, rebuild_if_missing,
             rc_params)
 
+    def find_fontsprop(self, prop, fontext='ttf', directory=None,
+                       fallback_to_default=True, rebuild_if_missing=True):
```

Comment:
Is the `fallback_to_default` argument used at all?

Reply:
I think I've covered most of it in the above comment, but to be specific:

Previously, `fallback_to_default` was useful since if a font family wasn't found, it would just fallback to default and be done with the parsing. (wouldn't touch the leftover families)

But that had to change, to at least parse every item in the list.